safe-room.lic

```diff
@@ -14,10 +14,15 @@ class SafeRoom
 
   def initialize
     arg_definitions = [
```

Comment:
Are there now two `arg_definitions = [` lines?

Reply:
It's a copy and paste error.